WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -32,7 +32,56 @@ def new
   end
 
   def index
-    @competitions = Competition.all.select([:id, :name, :cityName, :countryId]).order(:year, :month, :day).reverse_order
+    query = "CAST(CONCAT(year,'-',month,'-',day) as Datetime) > ? and showAtAll = true"
+    query_params = [(Date.today - 90)]
```

Comment:
Also, these variables look like they're only used in one place further down, I think it would be easier to read if you moved these next to where they're actually used.

I don't think `query_params` is a good name here, these are the params to query for "current" competitions, right? I think the names could be cleaner here. Also see my comments down below where you're doing the actual query.